zenodo/modules/utils/communities.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since this is more OpenAIRE-specific, `openaire.py` would be a better name for the file